src/main/resources/application.properties

```diff
@@ -1,6 +1,9 @@
-server.servlet.context-path: /item-handler
+server.servlet.context-path = /item-handler
 
-uk.gov.companieshouse.item-handler.health: /healthcheck
-uk.gov.companieshouse.item-handler.error-consumer: ${IS_ERROR_QUEUE_CONSUMER}
+uk.gov.companieshouse.item-handler.health = /healthcheck
+uk.gov.companieshouse.item-handler.error-consumer = ${IS_ERROR_QUEUE_CONSUMER}
 
-spring.kafka.consumer.bootstrap-servers: ${KAFKA_BROKER_ADDR}
\ No newline at end of file
+spring.kafka.consumer.bootstrap-servers = ${KAFKA_BROKER_ADDR}
+
+# TODO GCI-1097 Configure for Certs team
+certificate.order.confirmation.recipients = lmccarthy@companieshouse.gov.uk
```

Comment:
should leannes email be committed?

Reply:
Good question. The plan is to update the email address to that of the certs team in https://companieshouse.atlassian.net/browse/GCI-1097. But perhaps you are suggesting that email addresses are sensitive in general and we should look into providing them privately (through `chs-configs` or similar)?

Reply:
Yes that seems better. Also, if they wanted to change it, we would have to do a code change to do it. Also I notice it's only one, however does it allow sending to multiple recipients? If it doesn't is there a requirement to do it?

Reply:
I believe it does support multiple recipients, but I haven't tested that. It's a little bit theoretical with my test set up on Vagrant as it just goes to my MailHog SMTP server.

Reply:
@nabeelamirch In response to the first point I have now made changes so that the recipients will be provided via a `CERTIFICATE_ORDER_CONFIRMATION_RECIPIENTS` environment variable.

Reply:
@nabeelamirch In response to your other point, I have tested my assumption that multiple recipients are supported, and found it to be false! Whether I separate the email addresses with commas or semi-colons, I find that the CHS Notification API errors like this:

>   -> recipient: lmccarthy@companieshouse.gov.uk;nobody@nowhere.com
> 2020-06-08 14:04:22.873962697 +0100 BST m=+22491.735953672 [bZRjxAwnwekrBEmfdzdc] error: mail: expected single address, got ";nobody@nowhere.com"
>   -> error: mail: expected single address, got ";nobody@nowhere.com"
> 2020-06-08 14:04:22.874000051 +0100 BST m=+22491.735990975 [bZRjxAwnwekrBEmfdzdc] request
>   -> end: 2020-06-08 14:04:22.873997595 +0100 BST m=+22491.735988509
>   -> duration: 5.287861ms
>   -> status: 500
>   -> method: POST
>   -> path: /email
>   -> start: 2020-06-08 14:04:22.868709739 +0100 BST m=+22491.730700648

Reply:
You can leave it as it if needed. I'm happy to approve, you could allow it to have multiple recipients in the future.

Reply:
I will rename it as I see the story does only refer to a single email address in each instance. I will also raise a question with Emma about that being a potential requirement just in case (tomorrow).

Reply:
I have renamed that property and environment variable [here](https://github.com/companieshouse/item-handler/pull/10/commits/9fbd453f4070619b194fbc3a5301ee807faabedb) and in related PR https://github.com/companieshouse/chs-configs/pull/1053 too now.